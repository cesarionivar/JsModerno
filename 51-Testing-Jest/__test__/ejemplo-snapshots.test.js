const cliente = {
    nombre: 'Cesario Nivar',
    balance: 500,
    tipo: 'Premium'
};


describe('Testing al cliente', ()=>{ 
    test('Es Cesario', ()=> {
        expect(cliente).toMatchSnapshot();
    });
})