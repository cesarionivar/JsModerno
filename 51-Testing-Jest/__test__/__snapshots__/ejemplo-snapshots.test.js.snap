// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Cesario 1`] = `
Object {
  "balance": 500,
  "nombre": "Cesario Nivar",
  "tipo": "Premium",
}
`;
